controllers/communities.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 import json
```

Comment:
I only see json.load being used... so ...
```suggestion
from json import load as jload 
```

Reply:
Ah ok this makes sense! I wasnt aware that you can rename certain functions that you import so that they look better. In the case of the json lib, "loads" is a bit vague, so the rename helps!

Will do my best to keep this in mind!